src/Calypso-SystemTools-FullBrowser/ClyFullBrowserMorph.class.st

```diff
@@ -873,7 +873,7 @@ ClyFullBrowserMorph >> switchToVariables [
 { #category : 'taskbar-required' }
 ClyFullBrowserMorph >> taskbarLabel [
 
-	^ self newWindowTitle copyAfter: $:
+	^ self newWindowTitle copyAfter: Character space
```

Comment:
```suggestion
	| title |
	title := self newWindowTitle.
	^ title allButFirst: (title findString: ': ')
```

I would prefer to still check for the ':' in case at some point we have a title without the type prefix (I'm not sure that I managed all possible titles of Calypso and this code above has less risk of breaking the taskbar name)